src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/NewHandler.java

```diff
@@ -132,7 +134,8 @@ private boolean doesChainedMethodNeedsLineWrapping() {
             ast = ast.getParent();
         }
 
-        return TokenUtil.isOfType(ast, TokenTypes.ASSIGN, TokenTypes.LITERAL_RETURN);
+        return TokenUtil.isOfType(ast, TokenTypes.ASSIGN, TokenTypes.QUESTION, TokenTypes.COLON,
+            TokenTypes.LITERAL_RETURN);
```

Comment:
Does it make sense that these list of tokens is moved to a field since it is used in multiple places?

Reply:
ok, we can do that

Reply:
But the question is do these 2 areas always have to be the same? If they are reliant on both being exactly the same, then it should be made a field to make it easier for updates to occur to both places instead of only 1.